.github/workflows/test.yml

```diff
@@ -58,3 +58,5 @@ jobs:
                 run: cd ./fiberi18n && go test ./... -v -race
             -   name: Test fiberzerolog Middleware
                 run: cd ./fiberzerolog && go test ./... -v -race
+            -   name: Test jwt Middleware
+                run: cd ./jwt && go test ./... -v -race
```

Comment:
While we are at it, can you add go1.20 to the test matrix

Reply:
solved in websocket PR https://github.com/gofiber/contrib/pull/596